internal/watcher/watcher.go

```diff
@@ -356,7 +355,7 @@ func bumpVersion(ctx context.Context, path string) (err error) {
 	log.Infof(ctx, i18n.G("Bumping version for %s"), path)
 
 	// Windows-generated files do not have spaces around the equals sign.
-	if !ini.PrettyFormat {
+	if ini.PrettyFormat {
```

Comment:
hum, basically, you always sert it from True to False, why not doing it unconditionnally then? (the if is useless?)